templates/index.html

```diff
@@ -11,23 +11,115 @@
 
 -->
 <html>
-<head>
-	<style>
-	  html, body {
-        height: 100%;
-        margin: 0;
-        padding: 0;
-      }
-      #map {
-        height: 100%;
-      }
-	</style>
-	<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCUZzippHXKw9q2Il2OGD57Zz04CfxIrpY&signed_in=true&libraries=visualization"></script>
-	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
-	<script src="static/js/home.js"></script>
-</head>
+	<head>
+		<meta charset="utf-8">
+   	 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+    	<meta name="viewport" content="width=device-width, initial-scale=1">
+    	<meta name="description" content="">
+    	<meta name="author" content="">
+    	<title>TAAA</title>
+    	<!-- Bootstrap Core CSS -->
+    	<link href="static/css/bootstrap.min.css" rel="stylesheet">
+    	<!-- Custom CSS -->
+    	<link href="static/css/home.css" rel="stylesheet">
+    	<!--css for tabs-->
+    	<link href="static/css/tabs.css" rel="stylesheet">
+		<style>
+	 	 html, body {
+       		 height: 100%;
+        	 margin: 0;
+        	 padding: 0;
+      	}
+      	#map {
+        	height: 100%;
+      	}
+		</style>
+		<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCUZzippHXKw9q2Il2OGD57Zz04CfxIrpY&signed_in=true&libraries=visualization"></script>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
+		<script src="static/js/common.js"></script>
+
+	</head>
 	<body>
-		<div id="map">
-		</div>
+		<div class="container">
+			 <div class="row">
+        		<div class="col-md-3">
+        		</div>
+        		<div class="col-md-9">
+
+        		</div>
+			  </div>
+       		  <div class="row">
+				  <div class="col-md-3" id="facets">
+					  <div class="location-type">
+					  		<h4>Location Type</h4>
+              				<input type="radio" name="loc-type" value="0"> Intersection <br>
+                			<input type="radio" name="loc-type" value="1"> Segment <br>
+						    <input type="radio" name="loc-type" value=""> Both
+					  </div>
+              		  <div class="collision-type">
+						  <h4>Collision Type</h4>
```

Comment:
A complete list of collision type is here from HSIS, http://www.hsisinfo.org/guidebooks/north_carolina.cfm. 
For example, the value of head-on collision should be 27, value of pedestrian collision is 14. 

00 - Unknown
01 - Ran off Road Right
02 - Ran Off Road Left
03 - Ran Off Road Straight Ahead
04 - Jackknife
05 - Overturn/Rollover
06 - Crossed Centerline/Median
07 - Downhill Runaway
08 - Cargo/Equipment Loss or Shift
09 - Fire/Explosion
10 - Immersion
11 - Equipment Failure
12 - Separation of Units
13 - Other Non-Collision
14 - Pedestrian
15 - Pedalcyclist
16 - RR Train, Engine
17 - Animal
18 - Movable Object
20 - Parked Motor Vehicle
21 - Rear End, Slow or Stop
22 - Rear End, Turn
23 - Left Turn, Same Roadways
24 - Left Turn, Different Roadways
25 - Right Turn, Same Roadways
26 - Right Turn, Different Roadways
27 - Head On
28 - Sideswipe, Same Direction
29 - Sideswipe, Opposite Direction
30 - Angle
31 - Backing Up
32 - Other Collision with Vehicle
33 - Tree
34 - Military Route
35 - Luminaire Pole Non-Breakaway
36 - Luminaire Pole Breakaway
37 - Official Highway Sign Non-Breakaway
38 - Official Highway Sign Breakaway
39 - Overhead Sign Support
40 - Commercial Sign
41 - Guardrail End on Shoulder
42 - Guardrail Face on Shoulder
43 - Guardrail End on Median
44 - Guardrail Face on Median
45 - Shoulder Barrier End
46 - Shoulder Barrier Face
47 - Median Barrier End
48 - Median Barrier Face
49 - Bridge Rail End
50 - Bridge Rail Face
51 - Overhead Part Underpass
52 - Pier on Shoulder of Underpass
53 - Pier in Median of Underpass
54 - Abutment of Underpass
55 - Traffic Island Curb or Median
56 - Catch Basin or Culvert on Shoulder
57 - Catch Basin or Culvert on Median
58 - Ditch
59 - Embankment
60 - Mailbox
61 - Fence or Fence Post
62 - Construction Barrier
63 - Crash Cushion
64 - Other Fixed Object